### ISSUE TITLE
Error: Syntax error, unrecognized expression: unsupported pseudo: hover

### ISSUE THREAD

**Issue by AdamJamesWalters** [bbc/bbc-a11y#287]
## Summary
When running a11y on the following page;

```https://www.bbc.co.uk/newsround/46660407```

I get: 

```Unexpected Error Error: Syntax error, unrecognized expression: unsupported pseudo: hover```

and the tool doesn't run.

## Expected Behaviour
Tool should finish running.

## Current Behaviour
The tool fails with the following console output: 

```
Unexpected Error Error: Syntax error, unrecognized expression: unsupported pseudo: hover
    at Function.Sizzle.error (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:1541:8)
    at PSEUDO (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:1888:13)
    at matcherFromTokens (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2437:44)
    at Sizzle.compile (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2591:13)
    at Sizzle.select (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2677:16)
    at Sizzle (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:845:9)
    at Function.Sizzle.matchesSelector (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:1504:9)
    at Function.jQuery.filter (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2846:22)
    at winnow (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2835:16)
    at jQuery.fn.init.is (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2885:12)
Unexpected Error Error: Syntax error, unrecognized expression: unsupported pseudo: hover
    at Function.Sizzle.error (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:1541:8)
    at PSEUDO (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:1888:13)
    at matcherFromTokens (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2437:44)
    at Sizzle.compile (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2591:13)
    at Sizzle.select (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2677:16)
    at Sizzle (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:845:9)
    at Function.Sizzle.matchesSelector (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:1504:9)
    at Function.jQuery.filter (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2846:22)
    at winnow (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2835:16)
    at jQuery.fn.init.is (/Users/waltea07/workspace/accessibility-checker/node_modules/jquery/dist/jquery.js:2885:12)
0 pages checked, 0 errors found, 0 warnings, 0 standards skipped
```

~~~
~~~

## Steps to Reproduce (for bugs)

$ bbc-a11y https://www.bbc.co.uk/newsround/46660407

## Context & Motivation
Trying to check accessibility of BBC sites.

## Your Environment

* Version used: v2.2.6
* Operating System and version: Mac OSX 10.11.6 El Capitan
* Link to your project:

**Comment by ermi-ltd** [comment]
Hi,

In your css somewhere do you have:

“: hover” where’s there’s a space between the colon and the word hover?

I wonder if this is a typo tripping things up.

Cheers,

J&L

**Comment by ChrisBAshton** [comment]
Looks like a duplicate of #275.

**Comment by AdamJamesWalters** [comment]
No, there's no typos in the css

**Comment by marcusBBC** [comment]
Bitesize are also seeing this when running against a11y tests against their site.

FYI: @MazH-26

**Comment by EmmaJP** [comment]
Hi @AdamJamesWalters and @murkas , as Chris said this maybe similar to something that has come up before. @oluoluoxenfree said she is happy to help someone work on a solution for this. And Michael offered some advice about using a try-catch in #275 .

**Comment by murkas** [comment]
Cheers for nudging this @EmmaJP, Michael's suggestion of a `try...catch` will probably be the best solution, as it won't degrade the experience of using a11y, as far as I can tell.

**Comment by murkas** [comment]
I've had a quick look into this, this morning and have struggled to recreate (I guess CSS must have changed since this was initially report). Can anyone provide a link to a page that still causes this error? Thanks!

**Comment by greenc05** [comment]
@murkas yes I've just got this error when running bbc-a11y against https://www.bbc.co.uk/news/politics/constituencies/E14000939

Unexpected Error Error: Syntax error, unrecognized expression: [type="range"]:disabled::-webkit-slider-runnable-track
    at Function.Sizzle.error (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:1560:8)
    at Sizzle.tokenize (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2216:11)
    at Sizzle.compile (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2610:12)
    at Sizzle.select (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2700:16)
    at Sizzle (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:862:9)
    at Function.Sizzle.matchesSelector (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:1523:9)
    at Function.jQuery.filter (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2869:22)
    at winnow (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2858:16)
    at jQuery.fn.init.is (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2908:12)
    at getElementFontSizes (/usr/local/lib/node_modules/bbc-a11y/lib/standards/tests/textMustBeStyledWithUnitsThatAreResizableInAllBrowsers.js:43:28)
Unexpected Error Error: Syntax error, unrecognized expression: [type="range"]:disabled::-webkit-slider-runnable-track
    at Function.Sizzle.error (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:1560:8)
    at Sizzle.tokenize (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2216:11)
    at Sizzle.compile (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2610:12)
    at Sizzle.select (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2700:16)
    at Sizzle (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:862:9)
    at Function.Sizzle.matchesSelector (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:1523:9)
    at Function.jQuery.filter (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2869:22)
    at winnow (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2858:16)
    at jQuery.fn.init.is (/usr/local/lib/node_modules/bbc-a11y/node_modules/jquery/dist/jquery.js:2908:12)
    at getElementFontSizes (/usr/local/lib/node_modules/bbc-a11y/lib/standards/tests/textMustBeStyledWithUnitsThatAreResizableInAllBrowsers.js:43:28)

**Comment by murkas** [comment]
@greenc05 Thanks for sending this on - I'll investigate and see if I can determine where the error is thrown using this!

**Comment by andymsuk** [comment]
Hi @AdamJamesWalters and @murkas 
I've just released v2.4.1 which should fix this issue. Please re-open this issue if not.
Thanks.

**Comment by EmmaJP** [comment]
Thank you
